Modules/MedicalModule/Nursery/NurseryTalkScene.gd

```diff
@@ -39,8 +39,17 @@ func _run():
 		else:
 			saynn("You approach the nurse and try to get her attention. She drags her gaze away from the screen and looks at you.")
 
+			# (if player has the perk FertilityProudMom and didn't get an apple today)
+			if(GM.pc.hasPerk(Perk.FertilityProudMom) && !GM.main.getModuleFlag("MedicalModule", "Nursery_hadFreeAppleFromPerkToday", false)):
+				saynn("[say=nurse]Oh it's you again! I try to make sure that every mom here stays on the healthy side, take this[/say]")
+				saynn("She gives you an apple")
+				
+				GM.pc.getInventory().addItem(GlobalRegistry.createItem("appleitem"))
```

Comment:
this could be something like
```
var apple = GlobalRegistry.createItem("appleitem")
apple.setNotStolen(true)
GM.pc.getInventory().addItem(apple)
```

Reply:
Oh, btw, its a bad idea to do this in the _run function. Ideally the run function shouldn't change the state of the game ever. I'm a little guilty of that but I try to keep to only flag changes. Giving an apple is changing the state, even if it shouldn't break anything. How to solve that? Add a `_reactInit` function that gives the apple and also sets some variable like `gaveApple`. Then you just check that variable here in the run function